Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -76,60 +77,31 @@ class Navigator {
      Restrict direct initializer access.
      */
     private init() {
-        let settingsProfile = SettingsProfile(application: ProfileApplication.kMobile,
```

Comment:
Would it make sense to rebase current changes onto https://github.com/mapbox/mapbox-navigation-ios/pull/3010? I think I've updated shared navigator there as well to accomodate new API changes.